templates/buildkite-elastic.yml

```diff
@@ -127,14 +128,23 @@ Parameters:
 
   Subnets:
     Type: CommaDelimitedList
-    Description: Optional - Comma separated list of two existing VPC subnet ids where EC2 instances will run. Required if setting VpcId.
+    Description: Optional - Comma separated list of existing VPC subnet ids where EC2 instances will run. Required if setting VpcId.
     Default: ""
 
   AvailabilityZones:
     Type: CommaDelimitedList
     Description: Optional - Comma separated list of AZs that subnets are created in (if Subnets parameter is not specified)
     Default: ""
 
+  NetworkConfiguration:
+    Type: String
+    Description: Whether to configure your subnets as public, so your instances get public IP addresses (no extra cost) or private with redundant NAT instances ($58/month)
```

Comment:
I'm presuming this $58 is probably dependant on which region you launch into?

Reply:
Yeah, it's probably misleadingly specific. "Pricing starts at $0.045 per NAT gateway hour plus data processing and data transfer charges."